templates/scripts.tmpl/clean.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Unofficial strict mode
+set -euo pipefail
+IFS=$'\n\t'
+
+# Usage
+# npm run clean
+
+rm -rf dist/
```

Comment:
We shouldn't need this anymore, but we do need `coverage/`. Also prefer alphabetical (can still be directories first).